### PR TITLE
Fixes serum bug and adds clarifications to lychanthropy and draculaculiasis

### PR DESCRIPTION
**Pull request by DimWhat**
[Bug] [Secret]
## About the PR 
Fixes a massive exploit in the werewolf serum's code, fixes a runtime in the vampire disease, and adds some clarification to the lychanthropy and draculaculiasis transformation.

## Why's this needed? 
1. Bugs and exploits stink.
2. Clarification that people aren't antags when they become a werewolf or vampire is a good idea

### CODE REVIEW

File: code/datums/abilities/werewolf.dm [hunk 1]
```diff
@@ -97,7 +97,7 @@
 
 			M.visible_message("<span class='alert'><B>[M] [pick("metamorphizes", "transforms", "changes")] into a werewolf! Holy shit!</B></span>")
 			if (M.find_ailment_by_type(/datum/ailment/disease/lycanthropy))
-				boutput(M, "<span class='notice'><h3>You are now a werewolf.</span></h3>")
+				boutput(M, "<span class='alert'><h2>You are not an antagonist!</h2> Your transformation was achieved by in-game means, you have the form of a werewolf but are <i>not</i> an antagonist.</span></h3>")
```
Comment (PrimeNumb) [inline code/datums/abilities/werewolf.dm:100]: They might still be an antagonist that used the serum.

**Comment by ZeWaka** [comment]
:feelsgood:

File: code/datums/abilities/werewolf.dm (hunk 1 above)
Comment (DimWhat) [reply code/datums/abilities/werewolf.dm:100]: Yeah I kinda just copied the vampire one, didn't really think about this.

Comment (ZeWaka) [reply code/datums/abilities/werewolf.dm:100]: X

**Review by ZeWaka — CHANGES_REQUESTED** [review]
X

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**